Sources/client/Socket.swift

```diff
@@ -147,51 +150,36 @@ public class Socket {
   //----------------------------------------------------------------------
   // MARK: - Initialization
   //----------------------------------------------------------------------
+  public convenience init(_ endPoint: String) {
+    self.init(endPoint: endPoint,
+              transport: { url in return WebSocket(url: url) },
+              params: { nil })
+  }
+
+  @available(*, deprecated, message: "Deprecated in favour of Socket(_: String, paramsClosure: PayloadClosure?) instead")
```

Comment:
All of this can be deleted. The optional `params: Payload? = nil)` has the same effect as making a new `convenience init`